src/Microsoft.ServiceFabric.Actors/Runtime/ActorStateProviderHelper.cs

```diff
@@ -496,6 +496,99 @@ internal static IActorStateProvider CreateDefaultStateProvider(ActorTypeInformat
             return Task.FromResult(actorQueryResult);
         }
 
+        internal Task<PagedResult<ActorId>> GetStoredActorIdsForKvsAsync(

```

Comment:
Move this method to KVSActorStateProvider(Base).cs as this is no more a common code.

Reply:
Updated